### ISSUE TITLE
[Dashboard] Allow changing the filters with sensible user/project defaults

### ISSUE THREAD

**Issue by stdedos** [cypress-io/cypress#20349]
### What would you like?

I would like to be able to set personal and project-wide "Filter by" filter defaults.

### Why is this needed?

For a newly-started project, the default options of "Last 3 months" and "Weekly" make little sense, as we cannot see meaningful progress e.g. day-to-day

Filters are also "epheral" in the sense that, if I want to "preserve" a filter, I need to copy a URL that's ~750% longer than the original (but at least it's somewhat user-editable?)

**Comment by stdedos** [comment]
_Cypress should've already verified this within that 180-period_

**Comment by nagash77** [comment]
Hi @stdedos , again sorry that these tickets were not labeled in a way that would tell our stalebot to ignore them.  I have updated the labels accordingly.

**Comment by jennifer-shehane** [comment]
I agree with this feedback myself actually! This feedback is tracked within the product team for the Cloud. 

Closing here since this public repo is for tracking Cypress app specific issues / features.
